### PR TITLE
Rails generating files

### PR DESCRIPTION
**Pull request by kevkev300**
generating the models and db schema

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <%#= stylesheet_pack_tag 'application', media: 'all' %> <!-- Uncomment if you import CSS in app/javascript/packs/application.js -->
   </head>
   <body>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
```
Comment (marti15) [inline app/views/layouts/application.html.erb]: I would `<%= render 'shared/flashes' %>` in this file and then make a _flashes.html.erb to write the following code:
`<% if notice %>
  <div class="alert alert-info alert-dismissible" role="alert">
    <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
    <%= notice %>
  </div>
<% end %>
<% if alert %>
  <div class="alert alert-warning alert-dismissible" role="alert">
    <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
    <%= alert %>
  </div>
<% end %>`

File: db/migrate/20191202165358_create_offers.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CreateOffers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :offers do |t|
+      t.string :currency
+      t.integer :amount
+      t.integer :interes
```
Comment (marti15) [inline db/migrate/20191202165358_create_offers.rb]:
```suggestion
      t.integer :interest
```

File: db/schema.rb [hunk 3]
```diff
@@ -10,9 +10,52 @@
...
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["offer_id"], name: "index_loans_on_offer_id"
+    t.index ["user_id"], name: "index_loans_on_user_id"
+  end
+
+  create_table "offers", force: :cascade do |t|
+    t.string "currency"
+    t.integer "amount"
+    t.integer "interes"
```
Comment (marti15) [inline db/schema.rb]:
```suggestion
    t.integer "interest"
```

**Review by marti15 — CHANGES_REQUESTED** [review]
Looking nearly perfect herr Liebholz, just a tiny refactoring and a little mistake that needs addressing ;) and then we can merge it!

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (kevkev300) [reply app/views/layouts/application.html.erb]: should be refactored correctly

File: db/migrate/20191202165358_create_offers.rb (hunk 2 above)
Comment (kevkev300) [reply db/migrate/20191202165358_create_offers.rb]: changed and had do db:drop db:create and db:migrate

File: db/schema.rb (hunk 3 above)
Comment (kevkev300) [reply db/schema.rb]: got changed by changing and migrating the migration file above

**Pull request merged**